HorizontalPodAutoscaler/README.md

```diff
@@ -0,0 +1,26 @@
+See:
+https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/
+https://cloud.google.com/kubernetes-engine/docs/how-to/horizontal-pod-autoscaling
+
+To create the Deployment, apply the nginx.yaml manifest:
+> `kubectl apply -f nginx.yaml`
+
+To get a list of HPA objects in the cluster, use the following command:
+> `kubectl get hpa`
+
+To view an HPA's configuration and statistics, use the following command:
+> `kubectl describe hpa %hpa-name%`
```

Comment:
it might be better to put the name used in this specific example; users will want an unambiguous example to follow/copy